src/components/RenderRouter/ArchiveItem.tsx

```diff
@@ -0,0 +1,269 @@
+                    <div className="ArchiveItemEpisodeNum" >{this.props.content.showEpisodeNumbers === false ? null : item.episodeNumber + '. '} {item.episodeTitle}</div>
+                    <div className="ArchiveItemSeriesTitle">{item.seriesTitle != null ? item.seriesTitle : null}</div>
+                    <div className="ArchiveItemPublishedDate">{item.publishedDate}</div>
+                </div>
+            </div>);
+    }
+
+    renderSeries(item: SeriesByTypeData) {
+        if (!item) {
+            return null;
+        }
+        const seriesEnded = item.endDate && format(new Date(), 'yyyy-MM-dd') > item.endDate;
+        const videos = item.videos?.items ?? [];
+        if (videos.length > 0) {
+            return (
+                <div onClick={() => this.handleClick(videos.sort((a, b) => {
+                    const aNumber = a?.episodeNumber ?? 0;
+                    const bNumber = b?.episodeNumber ?? 0;
+                    return seriesEnded ? aNumber - bNumber : bNumber - aNumber;
+                })[0])}
+                    key={item.id} className="ArchiveItemVideo">
+                    <img alt={item.title + ' series image'}
+                        className="ArchiveItemImage2"
+                        src={'/static/photos/series/' + item.seriesType + '-' + (item.title ?? '').replace('?', '') + '.jpg'}
+                        onError={this.fallbackToImage('/static/NoCompassionLogo.png')}
+                    />
+                    <div className="ArchiveItemName" >{item.title}</div>
+                    <div className="ArchiveItemYearEpisode">{this.showYears(item.startDate, item.endDate)}{videos.length} {videos.length === 1 ? 'Episode' : 'Episodes'}</div>
+                </div>
+            );
+        }
+        else {
+            return null;
+        }
+    }
+
+    render() {
+        if (this.props.data && !this.state.currentNextToken && this.state.listData.length > 0) {
+            const videoData = this.state.listData as VideoByVideoTypeData[]
+            const seriesData = this.state.listData as SeriesByTypeData[]
+
+            switch (this.props.content.class) {
+                case 'series':
+                    const binnedSeriesData = this.binSeriesByYear(seriesData);
+                    return (
+                        <div className="ArchiveItemDiv1">
+                            {this.renderYearSelector(binnedSeriesData.years)}
+                            {binnedSeriesData.binned.filter(i => this.state.selectedYear === 'All' ? true : i[0]?.startDate?.slice(0, 4) === this.state.selectedYear.toString()).map((item, index) => {
+                                return (
+                                    <div key={index} >
+                                        <h1 className="ArchiveItemH1">{item[0]?.startDate?.slice(0, 4)}</h1>
+                                        <div className="ArchiveItemGrid">
+                                            {item.map(item2 => {
```

Comment:
Can you assign this a variable so you can use same markup for both? This seems to be the only difference other than the startData vs publishedDate, which can be fixed by using a "selector" function like I mentioned above.